src/App.js

```diff
@@ -1,7 +1,72 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
+import jwt_decode from 'jwt-decode';
+import Parse from 'parse';
 
-const App = () => {
-  return <h1>Hello React!</h1>;
-};
+function App() {
+  const [ user, setUser ] = useState({});
 
-export default App;
+  function handleCallbackResponse(response) {
+    console.log("Encoded User Token: " + response.credential);
+    var userObject = jwt_decode(response.credential);
+    console.log(userObject);
+    setUser(userObject);
+    document.getElementById("signInDiv").hidden = true;
```

Comment:
why are we hiding an element like this? Can't this be conditionally rendered based on state?